pkg/controllers/deprovisioning/helpers.go

```diff
@@ -209,6 +209,10 @@ func buildNodePoolMap(ctx context.Context, kubeClient client.Client, cloudProvid
 		if err != nil {
 			return nil, nil, fmt.Errorf("listing instance types for %s, %w", np.Name, err)
 		}
+		if len(nodePoolInstanceTypes) == 0 {
+			logging.FromContext(ctx).With("provisioner", np.Name).Debug("failed to get instance types for provisioner")
```

Comment:
```suggestion
			logging.FromContext(ctx).With("provisioner", np.Name).Debug("skipping provisioner, no resolved instance types found")
```

Is this a failure? Failed makes this sound like it should be an error